pkgs/top-level/python-packages.nix

```diff
@@ -10985,14 +10985,14 @@ in {
   };
 
   nototools = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

Reply:
Done - actually not to that specific location because it's not a library, it's a set of scripts used to assemble one specific font family - but done.